### DISCUSSION TITLE
Saving JSON files without annotated shapes directly?

### DISCUSSION THREAD

**Discussion by christian-post** [wkentaro/labelme#1249] [Ideas / Feature requests]
If I have a frame without any target classes to annotate, but want to save a JSON for it anyway, labelme would not let me save unless I create a shape with a label, and then immediately delete it. Only **Then** I can save the JSON with an empty shapes array. 

Is this intended? This is quite tedious.

**Comment by wkentaro** [comment]
@christian-post Yeah, this is intended behavior. If user opened an image but didn't do anything, the JSON file is not created.
However, I can imagine the case where you're frustrated, so maybe we can add an optional flag to enable "creating JSON file when a user open an image".
